### ISSUE TITLE
Написать правила сообщества

### ISSUE THREAD

**Issue by wsnark** [defcon-nn/defcon-nn.github.io#12]
Правила поведения для участников, докладчиков и hr'ов.

**Comment by GnuriaN** [comment]
HR - Доступ воспрещен 

![HR - Доступ воспрещен](http://vca-tuva.ru/sites/default/files/styles/large/public/images/1995.jpeg)
---

В качестве примера можно рассмотреть (спереть) [rannts.ru](http://rannts.ru/info/#rules). Там очень простые, но в тоже время понятные правила. Но они в основном распространяются на митапы, а не на сообщество.

**Comment by wsnark** [comment]
Да, в целом адекватно, хотя много букв наверно. Там кстати HRам наоборот свободный доступ - но с ограничениями.

**Comment by GnuriaN** [comment]
Оставим эйчаров в покое =) 

Я правильно понимаю, что данный профиль создан как "компания". Тогда почему бы организаторам сразу туда не добавится? А то намерения "открытость", а на деле "нет ни кого тут" =)

**Comment by GnuriaN** [comment]
А еще установить логотип 
[![Эмблема хакеров](https://upload.wikimedia.org/wikipedia/commons/thumb/4/45/Glider.svg/480px-Glider.svg.png)](https://ru.wikipedia.org/wiki/Эмблема_хакеров)
